src/wrapped_massa_db.rs

```diff
@@ -18,9 +12,11 @@ pub struct WrappedMassaDB(pub MassaDB);
 impl WrappedMassaDB {
     /// Returns a new `MassaDB` instance
     pub fn new(config: MassaDBConfig, convert_ledger_from_old_format: bool) -> Self {
-        let mut db_opts = Options::default();
-        db_opts.create_if_missing(true);
-        db_opts.create_missing_column_families(true);
+        let db_opts = Options::default();
+
+        // Note: no need to create anything (it can even be misleading if we specify the wrong path)
+        // db_opts.create_if_missing(true);
+        // db_opts.create_missing_column_families(true);
```

Comment:
I think we should add a bool argument to create it or not.
If we just load 1 DB to edit/read it, this change makes sence, but not if we want to convert an old_ledger to a new format. In that case, we would open the old_ledger with new(config, true, create_if_missing = false) and the new one with new(config, false, create_if_missing = true).

Reply:
Added --create-if-missing cmd line argument